src/SelectField/SelectField.jsx

```diff
@@ -1,4 +1,4 @@
-import CircularProgress from 'material-ui/CircularProgress'
+import CircularProgress from '../CircularProgress'
```

Comment:
`SelectField` only displayed the circle and no text, so there has to be a change to pass in an empty string or null or something for hte text